doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -92,6 +92,155 @@ writeShellScript "evaluate-my-file.sh" ''
 ```
 ::::
 
+### `makeDesktopItem` {#trivial-builder-makeDesktopItem}
+
+Write an [XDG desktop file](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) to the Nix store.
+
+This function is most often used with `mkDerivation.desktopItems` to add desktop items to a package.
+
+`makeDesktopItem` takes an attribute set according to the [XDG specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys), with the following possible values:
+
+Note that the 'Version' and 'Hidden' fields are not supported.
```

Comment:
I think it would be important to document that "Version" is actually hardcoded to the version that `makeDesktopItem` satisfies.